static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -154,14 +155,25 @@ const getMessageFromBraze = async (apiKey: string, brazeUuid: string): Promise<b
             return;
         }
 
-        appboy.subscribeToInAppMessage((message: InAppMessage) => {
+        let subscriptionId: ?string;
```

Comment:
I'm a bit confused about the scoping here. This line introduces a variable declaration but it's declared in the outer scope?

Reply:
Ah no, I was misreading the indentation, apologies.